supplementary_style_guide/glossary_terms_conventions/general_conventions/g.adoc

```diff
@@ -122,6 +122,16 @@ When referring to the command, use pass:q[`g++`], marked up appropriately.
 
 *See also*: xref:gdb[GDB], xref:insight[Insight]
 
+[[geo-replication]]
+==== image:images/caution.png[with caution] geo-replication (noun)
+*Description*: 1) In Red Hat Gluster Storage and related solutions, geo-replication refers to asynchronous replication of data in a Gluster file system from one location to another across a network. 2) In Red Hat Quay, geo-replication refers to replication of data from one central storage engine to other storage engines.
```

Comment:
I think this entry should be a bulleted list. See my similar comment earlier in this PR>
```suggestion
*Description*: 1) In Red Hat Gluster Storage and related solutions, _geo-replication_ refers to asynchronous replication of data in a Gluster file system from one location to another across a network. 2) In Red Hat Quay, _geo-replication_ refers to replication of data from one central storage engine to other storage engines.
```